lib/Net/SAML2/Binding/SOAP.pm

```diff
@@ -152,15 +153,25 @@ sub handle_response {
     my ($self, $response) = @_;
 
     my $saml = _get_saml_from_soap($response);
-    $self->verify_xml(
-        $saml,
-        no_xml_declaration => 1,
-        cert_text          => $self->idp_cert,
-        cacert             => $self->cacert,
-        anchors            => $self->anchors
-    );
-    return $saml;
-
+    my @errors;
+        my $success = try {
```

Comment:
I don't understand the indenting on this bit. I think it should be in a loop of sorts?